modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectLayoutResourceImpl.java

```diff
@@ -292,7 +294,28 @@ private com.liferay.object.model.ObjectLayoutRow _toObjectLayoutRow(
 	}
 
 	private com.liferay.object.model.ObjectLayoutTab _toObjectLayoutTab(
-		long objectDefinitionId, ObjectLayoutTab objectLayoutTab) {
+			long objectDefinitionId, ObjectLayoutTab objectLayoutTab)
+		throws PortalException {
+
+		long objectRelationshipId = GetterUtil.getLong(
+			objectLayoutTab.getObjectRelationshipId());
+
+		if (objectLayoutTab.getObjectRelationshipExternalReferenceCode() !=
+				null) {
```

Comment:
When it is a String, we should validate using this method `Validator.isNull()`.
When it is an Object, we should use !=.